Library/Client.ts

```diff
@@ -44,8 +44,11 @@ class Client {
      * @param   name    A string to identify this event in the portal.
      * @param   properties  map[string, string] - additional data used to filter events and metrics in the portal. Defaults to empty.
      * @param   measurements    map[string, number] - metrics associated with this event, displayed in Metrics Explorer on the portal. Defaults to empty.
+     * @param   tagOverrides the context tags to use for this telemetry which overwrite default context values
```

Comment:
It probably should. Any thoughts on the right language there? I didn't find any good descriptions of what that contextObjects collection does, though I can take a crack at it, based on my understanding.